### ISSUE TITLE
Twig: Sandbox Information Disclosure

### ISSUE THREAD

**Issue by StaffNowa** [symfony/demo#963]
Hello, symfony demo have vulnerabilities

```
!!  1 packages have known vulnerabilities.
!!  
!!  twig/twig (v2.6.2)
!!  ------------------
!!  
!!   * [CVE-NONE-0001][]: Sandbox Information Disclosure
!!  
!!  [CVE-NONE-0001]: https://symfony.com/blog/twig-sandbox-information-disclosure
```
This can be solved upgrade composer.json file

```
 "require": {
        ...
        "twig/twig": "^2.7",
        ...
    },
```

and run
```
composer update
```

**Comment by yceruto** [comment]
See https://github.com/symfony/demo/pull/955

**Comment by javiereguiluz** [comment]
Closing as fixed by #955. Thanks.
